website/docs/r/compute_shared_vpc_host.html.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: "google"
+page_title: "Google: google_compute_shared_vpc_host"
+sidebar_current: "docs-google-compute-shared-vpc-host"
+description: |-
+ Allows setting a Google Cloud Platform project to be a Shared VPC Host.
+---
+
+# google\_compute\_shared\_vpc\_host
+
+Allows setting a Google Cloud Platform project to be a Shared VPC Host. For more information see
+[the official documentation](https://cloud.google.com/compute/docs/shared-vpc)
+and
+[API](https://cloud.google.com/compute/docs/reference/latest/projects/enableXpnHost).
+
+## Example Usage
+
+```hcl
+resource "google_shared_vpc_host" "host" {
+  project = "your-project-id"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - (Optional) The project ID. If not specified, uses the
```

Comment:
I am confused why this field is marked as optional.

Reply:
Eh, I figured it _could_ be optional since it'll just fall back to the provider project. But it is sort of a weird UX. I'm fine with just making it Required.